webapp/channels/src/components/interactive_dialog/interactive_dialog.tsx

```diff
@@ -166,15 +177,12 @@ export default class InteractiveDialog extends React.PureComponent {
             elements,
         } = this.props;
 
-        let submitText = (
+        const submitText = (
             <FormattedMessage
                 id='interactive_dialog.submit'
-                defaultMessage='Submit'
+                defaultMessage={submitLabel ?? 'Submit'}
```

Comment:
This is changing the behavior. This is saying that if we don't find the translation for `interactive_dialog.submit` you us the defaultMessage, that is `Submit` or now the  `submitLabel`, that is not what we want, we want to have that `submitLabel` being the `submitText`in any case, not based on the existence or not of the translation.

Reply:
![image](https://github.com/mattermost/mattermost/assets/93595710/10be442f-8067-4e9f-bffa-c352214182dc)